code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -244,13 +244,21 @@
 	sealing = 1
 
 	if(!seal_target && !suit_is_deployed())
-		wearer.visible_message("<span class='danger'>[wearer]'s suit flashes an error light.</span>","<span class='danger'>Your suit flashes an error light. It can't function properly without being fully deployed.</span>")
+		wearer.visible_message("<span class='danger'>[wearer]'s suit flashes an error light.</span>",
+								"<span class='danger'>Your suit flashes an error light. It can't function properly without being fully deployed.</span>")
```

Comment:
```suggestion
		wearer.visible_message(
			SPAN_DANGER("[wearer]'s suit flashes an error light."),
			SPAN_DANGER("Your suit flashes an error light. It can't function properly without being fully deployed.")
		)
```